content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,662 @@
+
+   return adjustedWorldPosition;
+   ```
+5. Open the **Editor** and you should see your tree's leaves bending slightly. If you don't, try opening the **Material Editor** and increasing all the parameters. 
+
+{{< video src="/images/atom-guide/vegetation-bending-tutorial/detailbendingtree.mp4" autoplay="true" loop="true" width="100%" muted="true" info="Video of tree detail bending." >}}
+
+### Add main bending
+The leaves are moving now, but the tree doesn't sway yet. We will now add main bending, the overall sway and movement that the whole tree experiences.
+
+1. Open `{your-project-path}\Materials\Types\VegetationBending_Common.azsli`.
+2. At the bottom, add the following code:
+   
+   ```
+   float3 MainBending(float3 objectSpacePosition, float4 bending)
+   {
+      float windX = bending.x;
+      float windY = bending.y;
+      float bendScale = bending.z;
+
+      // More bending occurs higher up on the object
+      float bendFactor = objectSpacePosition.z * bendScale;
+      bendFactor *= bendFactor; 
+
+      // Rescale the displaced vertex position with the original distance to the object's center
+      // to restrict vertex movement to minimize deformation artifacts
+      float len = length(objectSpacePosition); 
+      float3 newPos = objectSpacePosition;
+      newPos.xy += float2(windX, windY) * bendFactor;
+
+      return normalize(newPos) * len;
+   }
+   ```
+   Using the current position of the vertex (after it has been changed from the detail bending) and the bending determined by the wind, we can bend the tree as a whole.
+3. Add a call to our detail bending function under the wind function call in our `ProcessBending` function:
+   
+   ```
+   if (o_color_isBound) {
+      // Overall wind
+      float4 currentBending = SetUpWindBending(currentTime, worldPosition);
+
+      // Detail bending
+      float3 currentOutPosition = DetailBending(position, normal, color, currentTime, worldPosition, currentBending.w);
+
+      currentOutPosition = MainBending(currentOutPosition, currentBending);
+
+      adjustedWorldPosition = mul(objectToWorld, float4(currentOutPosition, 1.0));
+   }
+   ```
+5. Open the **Editor** and you should see your tree swaying with the leaves still bending. If you don't, try opening the **Material Editor** and increasing the wind parameters.
+
+Amazing, our tree now sways and reacts to wind! Try to place multiple trees and observe how the trees sway differently when close together versus farther away. Also, add some lighting to make the trees pop!
+
+{{< video src="/images/atom-guide/vegetation-bending-tutorial/tree3.mp4" autoplay="true" loop="true" width="100%" muted="true" info="Video of trees swaying." >}}
+
+### Add motion vectors
+Because our tree is moving, we can add a *motion vector pass* so we can choose to add motion blur or other effects. For example, we have a partially implemented [Temporal Anti-Aliasing (TAA)](../../../atom-guide/features/taa.md) feature which uses the motion vectors. 
```

Comment:
'we can add' -> 'we should add'

Also, I would remove the 'partially implemented'. It is fully functional, there are just a lot of quality improvements we could make to it. But if they want to know about that, they can read more in the TAA document link you have right here.